tests/InputFiles/FindHorizons/FindHorizons3D.yaml

```diff
@@ -0,0 +1,53 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: FindHorizons3D
+# Check: parse
+
+ResourceInfo:
+  AvoidGlobalProc0: false
+  Singletons:
+    AhA:
+      Proc: Auto
+      Exclusive: true
+
+DomainCreator:
+  Shell:
+    InnerRadius: 1.9
+    OuterRadius: 10.
+    InitialRefinement: 0
+    InitialGridPoints: [8, 8]
+    UseEquiangularMap: True
+    EquatorialCompression: None
+    WhichWedges: All
+    RadialPartitioning: []
+    RadialDistribution: [Logarithmic]
+    TimeDependence: None
+
+Importers:
+  VolumeData:
+    FileGlob: "KerrVolume*.h5"
+    Subgroup: "VolumeData"
+    ObservationValue: Last
```

Comment:
Will this work for both the XCTS vars and the GH vars?

Reply:
No, currently the executable will always read in ADM variables (lapse, shift, spatial metric, extrinsic curvature).

Reply:
How much work would it be to allow it to read in GH vars? I ask because the main use case I'd have for this is dump data from an evolution either just during a run or after a failure, and then do a horizon find on it to see what's going on after-the-fact.

I believe the BBH executable does something similar already, right? It can read in ADM or GH vars?

Reply:
I believe we can reuse everything from the BBH numeric data importer except for the action that receives the data and transforms it: https://github.com/sxs-collaboration/spectre/blob/develop/src/Evolution/Systems/GeneralizedHarmonic/Actions/NumericInitialData.hpp#L196
We have to replace it with an action that does the same thing in reverse (GH -> ADM vars). I'd like to defer this to a follow-up as well.